src/components/JobsPanelParameters/JobsPanelParameters.js

```diff
@@ -115,7 +119,10 @@ const JobsPanelParameters = ({
 
     setNewJobParameters({
       ...newJobTaskSpecObj.parameters,
-      [parametersState.newParameter.name]: +parametersState.newParameter.value
+      [parametersState.newParameter.name]: convertParamValue(
+        parametersState.newParameter.value,
+        parametersState.newParameter.valueType
+      )
```

Comment:
Instead of always converting the entered parameter value to a number, convert it to a value depending on the parameter type.